ui/app/AppLayouts/Communities/views/CommunitySettingsView.qml

```diff
@@ -191,7 +191,7 @@ StatusSectionLayout {
             pubsubTopicKey: root.community.pubsubTopicKey
 
             sendModalPopup: root.sendModalPopup
-            tokensModel: root.community.communityTokens
```

Comment:
If this model is no longer used, it should be removed from the panel definition too.

Reply:
Done.